libdnf-cli/output/repoqueryformat.cpp

```diff
@@ -41,11 +42,9 @@ static const std::unordered_map<std::string, Getter> NAME_TO_GETTER = {
     {"release", &libdnf::rpm::Package::get_release},
     {"arch", &libdnf::rpm::Package::get_arch},
     {"evr", &libdnf::rpm::Package::get_evr},
-    {"nevra", &libdnf::rpm::Package::get_nevra},
     {"full_nevra", &libdnf::rpm::Package::get_full_nevra},
-    {"na", &libdnf::rpm::Package::get_na},
     {"group", &libdnf::rpm::Package::get_group},
-    {"packagesize", &libdnf::rpm::Package::get_package_size},
+    {"downloadsize", &libdnf::rpm::Package::get_package_size},
```

Comment:
Terminology we use for package sizes is a bit confusing here.
We have `packagesize` used in `Package::get_package_size()`, then there is `downloadsize` used here for query format. And then there is `rpm` and `libsolv` with their own terminology...
Shouldn't we agree on one term and use it consistently everywhere?

Reply:
That sounds good.
Possible options could be:
`packagesize` - `libdnf::rpm::Package::get_package_size()`
`downloadsize` - `libdnf::rpm::Package::get_download_size()`

It seems libsolv is using `SOLVABLE_DOWNLOADSIZE` so I would be in favor of renaming the method to `get_download_size()`. Also it seems dnf4 used `Package.downloadsize`. 

Do you have any other ideas? Or which would you prefer?

Reply:
I'd prefer using `downloadsize`. Packagesize sounds a bit vague to me - it's not completely clear what size we are referring to.

Reply:
Ok, I will create a separate PR with that change if that is ok.
It needs to be changed on a bunch of places.